config-model/src/test/java/com/yahoo/vespa/model/application/validation/change/search/AttributeChangeValidatorTest.java

```diff
@@ -202,4 +209,20 @@ public void changing_hnsw_index_property_neighbors_to_explore_at_insert_requires
                                                   "Field 'f1' changed: change hnsw index property " +
                                                   "'neighbors-to-explore-at-insert' from '200' to '100'"));
     }
+
+    @Test
+    public void removing_paged_requires_override() throws Exception {
+        try {
+            new Fixture("field f1 type tensor(x[10]) { indexing: attribute \n attribute: paged }",
+                        "field f1 type tensor(x[10]) { indexing: attribute  }").
+                    assertValidation(newRestartAction(ClusterSpec.Id.from("test"),
+                                                      "Field 'f1' changed: add attribute 'huge'"));
```

Comment:
Consider changing this string to something indicating that is doesn't matter. Empty string would be just fine.

Reply:
Fixed - thanks!